README.rst

```diff
@@ -543,7 +544,6 @@ is as follows:
 
        #!/bin/bash
        set -ex
-       cd "$(dirname "$0")"
```

Comment:
why to remove it?  keeping it helps for such scripts to be "ready to deploy" as a cron job without any outside `cd`.

Reply:
I even wonder now if we should improve it to place chronic and flock inside the script so that then it could be fully self-sufficient.

Reply:
Because the sample cronjob already includes a `cd`.  Including another `cd` in the script is a bit confusing, and I would hope that anyone who removes the `cd` in the cron command would know to add it to the script.

Reply:
I think the flock should be outside the script so that the Git actions don't interfere with each other.